runtime/include/module.xml

```diff
@@ -25,4 +25,11 @@
 <module>
 	<artifact type="reference" name="j9include">
 	</artifact>
+	
+	<artifact type="target" name="generate_jvmti" all="false">
+		<commands>
+			<command type="all" line="m4 -D JAVA_SPEC_VERSION=$(VERSION_MAJOR) $(UMA_PATH_TO_ROOT)include/jvmti.h.m4 > $(UMA_PATH_TO_ROOT)/include/jvmti.h"/>
```

Comment:
This contributes to a makefile in the `include` directory, so the prefix `$(UMA_PATH_TO_ROOT)/` is not necessary.
Also add a final newline to this file.